### DISCUSSION TITLE
how to operate the script ...

### DISCUSSION THREAD

**Discussion by warhawk200** [engageub/InternetIncome#178] [General]
Yesterday when i started the script and turned the values of honeygain and chrome to true , honeygain started to work correctly but chrome got some errors , 

1) So my question is that to try to fix chrome by trying some fixes do we need to close all containers by running this command ( sudo bash internetIncome.sh --delete ) and or we use docker ps and close each container manually 

2) Also how do i add new service for example grass without disturbing old running containers  , because if we want to try any service then we need to run the --start script again which requires closing all running containers....

**Comment by engageub** [comment]
Hello,
Could you please be specific on what errors you got on chrome.

> * So my question is that to try to fix chrome by trying some fixes do we need to close all containers by running this command ( sudo bash internetIncome.sh --delete ) and or we use docker ps and close each container manually

If you know which container has problem, you can delete that particular container. You can also delete all containers by running `sudo bash internetIncome --delete` command. However you will have backup of chrome folders.  Use the `--deleteBackup` option to delete the backup folders. 


> 2\. Also how do i add new service for example grass without disturbing old running containers  , because if we want to try any service then we need to run the --start script again which requires closing all running containers....

You can create a new folder and download the InternetIncome code again. Then configure in `properties.conf` file and start the script from the newly created folder.
Please refer to [FAQ](https://github.com/engageub/InternetIncome/wiki/Frequently-Asked-Questions#-how-to-use-multiple-accounts).

Thank you

**Reply by warhawk200** [reply]
yes these are errors i am facing on oracle cloud aaarch64 , chrome

![image](https://github.com/user-attachments/assets/a33e7025-6a84-47ac-986c-722aec3a059c)
![image](https://github.com/user-attachments/assets/7baebdec-5943-49c8-8eeb-7e40736125df)

**Reply by engageub** [reply]
Hello,
There may be common errors or warnings in the container at the beginning. You will have to follow up with docker image owner to resolve those errors. However they may already be aware of the default errors. 
Were you able to access the chrome container in browser?

Thank you

**Reply by warhawk200** [reply]
okay , i am unable to access that page also , page says "ERR_CONNECTION_TIMED_OUT"

can you please link me there github .....

**Reply by engageub** [reply]
It is mentioned in the issue https://github.com/linuxserver/docker-chromium/issues/28 that they are Openbox warnings and they have no effect on the functionality of the container.

**Reply by warhawk200** [reply]
umm looks like page is now working i needed to open oracle network 
![image](https://github.com/user-attachments/assets/3f626b9d-72c6-40f9-a336-8b3eef44d8ed)
but i am getting black screen here

**Reply by warhawk200** [reply]
thanks for the link checking it out 😊

**Reply by engageub** [reply]
Sometimes it takes time. Can you right click on the screen and see if chrome option is available.
Also make sure your CPU has enough resources. 

Thank you

**Reply by engageub** [reply]
The following is one of the last comments from the issue mentioned. However, it is supposed to automatically start instead of manually opening the terminal and starting.
 
```

right click the desktop, open a terminal and run chromium see what it spits out. 
Use chromium --no-sandbox if you are running without seccomp unconfined.

```

**Reply by warhawk200** [reply]
tried both , typing chromium gives this log
![image](https://github.com/user-attachments/assets/d99ea65c-d51d-4c55-9f4c-c8f546ec2462)
and 

typing chromium --no-sandbox gives this log 

![image](https://github.com/user-attachments/assets/88631baa-d271-4192-ad6c-1003af063543)

**Comment by engageub** [comment]
The latest docker image has been updated 5 days back. 
An user already opened related issue 2 days back. 
Please see https://github.com/linuxserver/docker-chromium/issues/38
I just updated to latest version on my PC and have the same blank screen issue.

**Reply by warhawk200** [reply]
i think issue maybe related to oracle cloud maybe , because when i tried running same script on amd64 contabo server , it was working fine . But my contabo server is weak thats why i dont want to use this script on contabo

**Reply by warhawk200** [reply]
umm which version were you using earlier , is there a way to install and try that version?

**Comment by engageub** [comment]
Hello,
Can you replace `lscr.io/linuxserver/chromium:latest` to `lscr.io/linuxserver/chromium:version-02e790f2` in `internetIncome.sh` file and start.

Thank you

**Reply by engageub** [reply]
Please wait for a minute after starting the container for the browser to open

**Reply by engageub** [reply]
Tried again with latest version and is working fine. Just wait for a minute and then open the VNC browser. Do not open the browser immediately after starting the container.

**Reply by warhawk200** [reply]
i tried deleting everything and installing latest version again , still got same blackscreen 
but
then i tried the version you recommended , it worked 😁
![image](https://github.com/user-attachments/assets/50fea0e4-ff1f-4b49-8346-6e31673ea44a)
thanks for all the help 🙌
